ex1/index.js

```diff
@@ -0,0 +1,14 @@
+function get_sum(numere) {
+    let get_sum = 0;
```

Comment:
```suggestion
    let sum = 0;
```
In acest caz, la numele variabileleor, nu mai e nevoie de prefixul "get", se pune doar la functii